quodlibet/quodlibet/util/trash.py

```diff
@@ -121,9 +122,13 @@ def trash_free_desktop(path):
         raise
 
 
-def can_trash():
+def use_trash():
     """If the current platform supports moving files into a trash can."""
-    return (os.name == "posix" and sys.platform != "darwin")
+    if config.has_option("settings", "bypass_trash"):
+        bypass = config.getboolean("settings", "bypass_trash")
+    else:
+        bypass = False
+    return os.name == "posix" and sys.platform != "darwin" and not bypass
```

Comment:
One thing I considered and did not do yet, because it might be premature optimization, is move the bypass check into a separate method and call that as the third clause, so that we'd only have to do that work if the first two checks aren't False.